aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1689,7 +1696,12 @@ impl AptosVM {
 
         SYSTEM_TRANSACTIONS_EXECUTED.inc();
 
-        let output = VMOutput::new(change_set, FeeStatement::zero(), VMStatus::Executed.into());
+        let output = VMOutput::new(
+            change_set,
+            FeeStatement::zero(),
+            VMStatus::Executed.into(),
+            TransactionAuxiliaryData::default(),
```

Comment:
Btw, any particular reason why use default? To me it seems like a more explicit names like `none()` or `empty()` are always better? Otherwise you have to check the impl to see what it the default

Reply:
We will have more fields in the struct and more convenient to use an existing trait.